Test/Integration/Product/MultiStoreReplicaTest.php

```diff
@@ -63,6 +65,35 @@ public function testMultiStoreReplicaConfig()
         $this->resetAllSortings();
     }
 
+    public function testCustomerGroupsConfig()
+    {
+        $defaultStore = $this->storeRepository->get('default');
+        $fixtureSecondStore = $this->storeRepository->get('fixture_second_store');
+        $fixtureThirdStore = $this->storeRepository->get('fixture_third_store');
+
+        // Enable customer groups for second fixture store and save configuration
+        $this->setConfig( ConfigHelper::CUSTOMER_GROUPS_ENABLE, 1, $fixtureSecondStore->getCode());
+        $this->indicesConfigurator->saveConfigurationToAlgolia($fixtureSecondStore->getId());
+        $this->algoliaHelper->waitLastTask($fixtureSecondStore->getId());
+
+        // 7 indices for default store and third store:
+        // - 1 for categories
+        // - 1 for pages
+        // - 1 for suggestions
+        // - 4 for products (1 main and 3 for sorting replicas)
+        $this->assertEquals(7, $this->countStoreIndices($defaultStore));
+        $this->assertEquals(7, $this->countStoreIndices($fixtureThirdStore));
+
+        // 13 indices for second store:
+        // - 1 for categories
+        // - 1 for pages
+        // - 1 for suggestions
+        // - 10 for products (1 main and 9 replicas (2 prices sortings * 4 customer groups + 1 other sorting (created_at))
+        $this->assertEquals(13, $this->countStoreIndices($fixtureSecondStore));
+
+        $this->resetAllSortings();
```

Comment:
Is this still needed if we are also invoking from `tearDown()`?

Reply:
good catch, this is not needed indeed